src/gui/raster/qgssinglecolorrendererwidget.h

```diff
@@ -0,0 +1,54 @@
+/***************************************************************************
+                         qgssinglecolorrendererwidget.h
+                         ---------------------------------
+    begin                : April 2024
+    copyright            : (C) 2024 by Mathieu Pellerin
+    email                : mathieu at opengis dot ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSSINGLECOLORRENDERERWIDGET_H
+#define QGSSINGLECOLORRENDERERWIDGET_H
+
+#include "ui_qgssinglecolorrendererwidgetbase.h"
+
+#include "qgsrasterrendererwidget.h"
+#include "qgis_sip.h"
+#include "qgis_gui.h"
+
```

Comment:
We could SIP_NO_FILE this one, couldn't we?

Reply:
@nyalldawson , all other widgets are exposed, feels like we should match expectation.

Reply:
That's definitely just a historic oddity -- I can't see any reason they need to be exposed as stable API

Reply:
@nyalldawson , so, do we expose this now and unexpose all in QGIS 4.0, or do we start now by skipping this one here? :)

Reply:
We should start by not exposing new ones